### ISSUE TITLE
periventricular pseudocysts

### ISSUE THREAD

**Issue by yuval-yaron** [obophenotype/human-phenotype-ontology#7010]
**Preferred term label:**
periventricular pseudocysts

**Synonyms**
PVPC 
subependymal pseudo cysts
germinolytic cysts
connatal cysts
**Definition (free text, please give PubMed ID)**
pseudocysts that lack an epithelial lining, PVPCs are frequently observed in the caudothalamic groove or in the infero-lateral aspect of the frontal horns of the lateral ventricles, but may also be located elsewhere along the periventricular germinal matrix, as in the temporal or occipital horns PMID: 32277778

**Parent term (use hpo.jax.org/app)**
Abnormal periventricular white matter morphology HP:0002518

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Zellweger syndrome  PS214100
cytomegalovirus infection
**Your nano-attribution (ORCID)**
0000-0001-8622-7668

**Comment by patt1morgan** [comment]
@TonyRoscioli, @carlotarodo.   Another new term request which we would appreciate you taking the time to review for us.  All comments welcome! Thank you...

**Comment by pnrobinson** [comment]
prefered label: Periventricular pseudocyst
synoynm, plrual form Periventricular pseudocysts

note -- prognosis very dependent on the location (revised definition)

**Comment by pnrobinson** [comment]
adding term
